gradio/components/gallery.py

```diff
@@ -18,6 +19,10 @@
 set_documentation_group("component")
 
 
+GalleryImageType = Union[np.ndarray, _Image.Image, Path, str]
```

Comment:
for consistency:
```suggestion
GalleryImageType = np.ndarray | _Image.Image | Path, str
```

Reply:
Ah, this syntax is not allowed here.

![CleanShot 2023-12-09 at 13 59 04@2x](https://github.com/gradio-app/gradio/assets/3135397/22488905-2fbc-4b39-9f53-3cc789c5d1c4)

It also fails in the CI (https://github.com/gradio-app/gradio/actions/runs/7151314806/job/19475505359?pr=6487):
```
Run . venv/bin/activate
Traceback (most recent call last):
  File "js/_website/generate_jsons/generate.py", line 5, in <module>
    from src import changelog, demos, docs, guides
  File "/home/runner/work/gradio/gradio/js/_website/generate_jsons/src/docs/__init__.py", line 6, in <module>
    import gradio
  File "/home/runner/work/gradio/gradio/gradio/__init__.py", line 3, in <module>
    import gradio._simple_templates
  File "/home/runner/work/gradio/gradio/gradio/_simple_templates/__init__.py", line 1, in <module>
    from .simpledropdown import SimpleDropdown
  File "/home/runner/work/gradio/gradio/gradio/_simple_templates/simpledropdown.py", line 6, in <module>
    from gradio.components.base import FormComponent
  File "/home/runner/work/gradio/gradio/gradio/components/__init__.py", line 27, in <module>
    from gradio.components.gallery import Gallery
  File "/home/runner/work/gradio/gradio/gradio/components/gallery.py", line 22, in <module>
    GalleryImageType = np.ndarray | _Image.Image | Path, str
TypeError: unsupported operand type(s) for |: 'type' and 'type'
Error: Process completed with exit code 1.
```

Probably the PEP604 syntax is not allowed for defining type alias, not using as type hints?

Anyway I would revert it.